app/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\DB;
+
+use App\Contact;
+
+class ContactController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $contacts = DB::table('contacts')->get();
+
+        foreach ($contacts as $contact) {
+            try {
+                $response = file_get_contents(env('EMAIL_SUBSCRIPTION_API') . '/get_status?email=' . $contact->email);
+                $data = json_decode($response);
+                if ($data['status'] == 'subscribed') {
+                    $contact->subscribed = true;
+                } else {
+                    $contact->subscribed = false;
+                }
+            } catch (\Exception $e) {
+                Log::error($e);
+                $contact->subscribed = false;
+            }
+        }
+
+        return view('contacts.index', compact('contacts'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('contacts.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $contact = new Contact([
```

Comment:
[Bad] - In as much as it's okay to actually instantiate the model, we could have made use of one aspect of OOP called Trait in the controller to be able to use Model. Something like this
`use App\Models\Contact;`.